inc/saiswitch.h

```diff
@@ -1269,6 +1280,55 @@ typedef enum _sai_switch_attr_t
     SAI_SWITCH_ATTR_ACL_STAGE_EGRESS,
 
     /**
+     * @brief The number of lossless queues per port supported by the switch
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_QOS_NUM_LOSSLESS_QUEUES,
+
+    /**
+     * @brief Control for buffered and incoming packets on queue undergoing PFC Deadlock Recovery.
+     *
+     * This control applies to all packets on all applicable port/queues. If application wants finer packet
+     * action control on per port per queue level then it is expected to set this control to packet forward
+     * and install one or more ACL and enable/disable them in the DLD/DLR event callback
+     * (SAI_SWITCH_ATTR_PFC_DEADLOCK_EVENT_NOTIFY) respectively.
+     *
+     * @type sai_packet_action_t
+     * @flags CREATE_AND_SET
+     * @default SAI_PACKET_ACTION_DROP
+     */
+    SAI_SWITCH_ATTR_PFC_DLR_PACKET_ACTION,
+
+    /**
+     * @brief PFC Deadlock Detection timer interval in milliseconds.
+     *
+     * If the monitored queue is in XOFF state for more than this duration then
+     * its considered to be in a PFC deadlock state and recovery process is kicked off.
+     * Note: Use COS value as key and timer interval as value.
+     *
+     * @type sai_map_list_t
+     * @flags CREATE_AND_SET
+     * @default empty
+     */
+    SAI_SWITCH_ATTR_PFC_COS_DLD_INTERVAL_MAP_LIST,
```

Comment:
Removed the trailing MAP_LIST, but still seeing this error:

should this be CREATE_AND_SET?
 ASSERT FAIL(on line 828) SAI_SWITCH_ATTR_PFC_COS_DLD_INTERVAL: default value is required but this attr value type is not supported yet
Makefile:52: recipe for target 'all' failed
make: *** [all] Error 1

I think the default=empty is not working for this yet.